recipe/meta.yaml

```diff
@@ -12,16 +12,15 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1002
-  skip: true  # [not py2k]
+  number: 1003
+  noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
-  build:
-    - python
-
+  host:
+    - python >=3.5
   run:
-    - python
+    - python >=3.5
```

Comment:
Looking at CI it appears Python 3.5 is the oldest version supported, is that correct @Jhsmit? If so, this should respect that. If it should be a newer version, we can bump these as well.

Reply:
yes, python 3.5 and up.